spec/Section 4 -- Introspection.md

```diff
@@ -352,7 +352,7 @@ Fields\:
 
 **Input Object**
 
-Input objects are composite types defined as a list of named input values. They
+Input objects are types defined as a list of named input values. They
```

Comment:
I think the `composite` improves the clarity of the text. I think that the term "composite type" is fairly well understood in prose, e.g. [Wikipedia: composite data type](https://en.wikipedia.org/wiki/Composite_data_type). I suggest we revert this change.

```suggestion
Input objects are composite types defined as a list of named input values. They
```